src/components/layout/Live.js

```diff
@@ -0,0 +1,125 @@
+import { React, useState } from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import { Button, Typography } from "@material-ui/core";
+
+import PeopleOutlineIcon from "@material-ui/icons/PeopleOutline";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Mention styles in components only if dynamic values or style bindings are used. Make sure to have static styles to one or multiple css files